### PR TITLE
DAOS-7488 documentation: document usage of daos autotest in daos admi…

### PR DESCRIPTION
**Pull request by cdurf1**
DAOS-7488 documentation: document daos autotest in daos admin guide

Skip-build: true
Skip-test: true

Signed-off-by: craig <craig.durfey@intel.com>

### CODE REVIEW

File: doc/admin/deployment.md [hunk 1]
```diff
@@ -1053,42 +1053,63 @@ the `[Service]` section before reloading systemd and restarting the
...
+     97  Destroying container       OK      0.030
+     98  Disconnecting from pool    OK      0.010
+     99  Tearing down DAOS          OK      0.000
+
+    All steps passed.
+
+Note: The command is executed in a development environment,
+performance differences will vary, based on your system.
+Note 2: Smaller pools may show DER_NOSPACE(-1007): 'No space
+on storage target'
```
Comment (johannlombardi) [inline doc/admin/deployment.md:1111]: should probably be moved to !!! note

File: doc/admin/deployment.md [hunk 2]
```diff
@@ -1053,42 +1053,63 @@ the `[Service]` section before reloading systemd and restarting the
...
-```bash
-mpirun -np <num_clients> --hostfile <hostfile> ./daos_test
-```
 
-`daos_test` requires at least 8GB of SCM (or DRAM with tmpfs) storage on
-each storage node.
+A pool must be created to run the autotest:
+    
+    dmg pool create --size=100G --ranks=0
+
```
[the remark this answers is not in the document]
Comment (johannlombardi) [reply doc/admin/deployment.md:1070]: Please fix

File: doc/admin/deployment.md [hunk 3]
```diff
@@ -1053,42 +1053,63 @@ the `[Service]` section before reloading systemd and restarting the
...
-```
 ### Launch the client application
-```bash
-mpirun -np <num_clients> --hostfile <hostfile> ./daos_test
-```
 
-`daos_test` requires at least 8GB of SCM (or DRAM with tmpfs) storage on
-each storage node.
+A pool must be created to run the autotest:
+
```
[the remark this answers is not in the document]
Comment (johannlombardi) [reply doc/admin/deployment.md:1068]: Please fix.

File: doc/admin/deployment.md [hunk 4]
```diff
@@ -1053,42 +1053,63 @@ the `[Service]` section before reloading systemd and restarting the
...
-To manually configure the network interface, set `OFI_INTERFACE` such as:
-```
-export OFI_INTERFACE=lo
-```
-When manually configuring an Infiniband device with a verbs provider, the network
-device domain is required.  To manually configure the domain, set `OFI_DOMAIN` such as:
-```
-export OFI_DOMAIN=hfi1_0
-```
 ### Launch the client application
```
Comment (johannlombardi) [inline doc/admin/deployment.md:1065]: Sounds like we don't really need this sub-section any longer given that the text is shorter.

**Review by johannlombardi — CHANGES_REQUESTED** [review]
Looks good. Just a few changes.

**Comment by omar-ocampo** [comment]
Looks good to me

**Review by omar-ocampo — APPROVED** [review]

**Pull request closed without merging**